### PR TITLE
Improve downloading of logs

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by Etsija** [comment]
The UI part looks OK to me.

File: ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx [hunk 1]
```diff
@@ -214,7 +218,13 @@ const RunComponent = () => {
               <TableCell>Logs</TableCell>
               <TableCell>
                 <Button
-                  onClick={() => downloadLogs(ortRun.id)}
+                  onClick={() =>
+                    downloadLogs(
+                      ortRun.id,
+                      '',
+                      Object.keys(ortRun.jobs).concat('config').join(',')
```
Comment (mmurto) [inline ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx:225]: Does the core accept empty string `level` just fine?

Comment (mmurto) [reply ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx:225]: Apparently modified in separate commit to do so.

Comment (mmurto) [inline ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx:225]: I would either add a code comment here explaining why this is done or extract it into a descriptive variable.

**Review by mmurto — CHANGES_REQUESTED** [review]

**Review by mmurto — APPROVED** [review]

**Pull request merged**